model/textparse/promparse.go

```diff
@@ -251,6 +251,13 @@ func (p *PromParser) nextToken() token {
 	}
 }
 
+func (p *PromParser) errorPosition(err error) error {
+	str := string(p.l.b[:p.l.i])
+	line := 1 + strings.Count(str, "\n")
+	column := 1 + p.l.i - (strings.LastIndex(str, "\n") + len("\n"))
+	return errors.Errorf(err.Error()+" line %d, column %d", line, column)
```

Comment:
For clarity, I think it would be helpful to wrap the line and column numbers in parenthesis to distinguish from the actual error.

```suggestion
	return errors.Errorf(err.Error()+" (line %d, column %d)", line, column)
```